keyboards/romac/info.json

```
@@ -0,0 +1,13 @@
+{
+  "keyboard_name": "RoMac",
+  "url": "",
+  "maintainer": "qmk",
+  "width": 3,
+  "height": 4,
+  "layouts": {
+    "LAYOUT_ortho_3x4": {
+      "key_count": 12,
+	  "layout": [{"x":0, "y":0}, {"x":1, "y":0}, {"x":2, "y":0}, {"x":0, "y":1}, {"x":1, "y":1}, {"x":2, "y":1}, {"x":0, "y":2}, {"x":1, "y":2}, {"x":2, "y":2}, {"x":0, "y":3}, {"x":1, "y":3}, {"x":2, "y":3}, {"x":3, "y":3}]
```

You actually have 13 keys defined here, but `romac.h` specifies that there are 12:

https://github.com/qmk/qmk_firmware/blob/8b1e096bd12689fdb8bc290659995b71db8b102f/keyboards/romac/romac.h#L7-L10

These numbers should match for this file to work as intended.

```suggestion
	  "layout": [{"x":0, "y":0}, {"x":1, "y":0}, {"x":2, "y":0}, {"x":0, "y":1}, {"x":1, "y":1}, {"x":2, "y":1}, {"x":0, "y":2}, {"x":1, "y":2}, {"x":2, "y":2}, {"x":0, "y":3}, {"x":1, "y":3}, {"x":2, "y":3}]
```